### ISSUE TITLE
Proposal: Built-in vpn client

### ISSUE THREAD

**Issue by kirillDanshin** [rancher/os#1226]
I propose to add a vpn client in rancher OS.
In my setup I want to use rancher hosts without an external IP address, but docker hub requires internet connection.
Can we embed openvpn or something like this in the distro?

**Comment by SvenDowideit** [comment]
I'd like that too :)

**Comment by pheman** [comment]
Yes, it's especially important when network security is considered.
